### ISSUE TITLE
Restructure folders so that all sidebar panel code is in one place

### ISSUE THREAD

**Issue by markerikson** [replayio/devtools#8794]
there's some things about our current folder structure and naming that I hate:

- `/SecondaryToolbox` as a name for "the stuff that goes in the tabbed panel like 'Console' and 'Elements' " is unintuitive and I keep forgetting that that lives in `src/ui/components/SecondaryToolbox/`
- the `<SidebarPanel>` component is in `src/ui/components/SidePanel.tsx`
- the panels for "Replay Info", "Test Suites", "Protocol Viewer", and "Comments" are in that same folder
- the sidebar panels, on the other hand, are over in `src/devtools/client/debugger/src/components/`
- the Sources Tree is in `/PrimaryPanes/` (????)
- the "Pause" panel is in `/SecondaryPanes/` (???!?!!?!?)

in other words, they're split into two totally separate areas, and the naming doesn't actually tell me what these things are

It would be nice if all the sidebar panels lived in one parent folder, and we had better names for the "Sources" panel and "Pause" panel than "Primary" and "Secondary".

(For that matter, `src/ui/components` has gotten kinda large and grab-bag-y, and could stand to be split up into more folders).

Internal issue: FE-1226

**Comment by melencholicmice** [comment]
Hello , I would like to work on this issue.

**Comment by jasonLaster** [comment]
@melencholicmice go for it

**Comment by anikdhabal** [comment]
So can I put all the sidebar panel code inside a folder in src/ui/components? @jasonLaster @markerikson

**Comment by jasonLaster** [comment]
@anikdhabal i'll defer to mark

**Comment by anikdhabal** [comment]
okk
